west.yml

```diff
@@ -163,7 +163,7 @@ manifest:
         - hal
     - name: hal_bouffalolab
       path: modules/hal/bouffalolab
-      revision: c6c44b879503d990dfba643c212b606cee414faa
+      revision: 2e9606caf08fba2c1054093ad672a8193d45373e
```

Comment:
You should use head/pull/1 till we got green CI and then hal PR got merged. In this case, any updates on the hal PR will be reflected here.

Reply:
done